packages/node_modules/@webex/internal-plugin-conversation/src/decryption-transforms.js

```diff
@@ -152,9 +152,14 @@ export const transforms = toArray('inbound', {
     if (conversation.avatarEncryptionKeyUrl) {
       promises.push(ctx.transform('decryptObject', conversation.avatarEncryptionKeyUrl, conversation.avatar));
     }
+    // TODO (holsted 04/06/19): This was deprecated in favor of .previousValue below. I wanted to remove this entirely
+    // but I wasn't sure if some open source use cases may be reading from cached conversations or not so leaving it for now.
```

Comment:
This works and allows for the use case defined. Good thinking.

Reply:
Could we override the `conversation.previous` getting and add a logger statement to indicate that it is deprecated (as well as return what is in `.previousValue`)?

Reply:
Sure. I can update this

Reply:
On second though @adamweeks - no consumer should be using .previous since it's a transform. I think we should leave this as is and remove in the future.

Reply:
@andrewholsted sounds good